packages/namekit-react/src/components/Checkbox.tsx

```diff
@@ -0,0 +1,15 @@
+import cc from "classcat";
+import React, { InputHTMLAttributes } from "react";
+
+export const Checkbox = (props: InputHTMLAttributes<HTMLInputElement>) => {
```

Comment:
I shared other comments on this PR related to making the label associated with the <input element somehow part of the code for this new `<Checkbox ...` component.

What do you think about making it so that we make it so that the content of the label is passed as children to `<Checkbox ...`? The goal in sharing this suggestion is to try to keep apps that use our `Checkbox` component the freedom to fully customize the styling or other details of the label without making the params for `Checkbox` really complicated. Maybe this is a flawed idea I'm sharing? Appreciate your advice :+1:

Reply:
@lightwalker-eth Sounds good! Done it. The only thing that might be confusing is that we're not directly telling the user it will be rendered as a label, but I think it’s fine since it simplifies things